Makefile.common

```diff
@@ -54,6 +54,7 @@ misspell-correction: $(TOOLS_BIN_DIR)/misspell
 gofmt: $(GOFUMPT) $(GOIMPORTS) misspell-correction
 	gofmt -w -s .
 	$(GOFUMPT) -l -w .
+	$(MAKE) gogci
```

Comment:
This writes changes for import ordering when they're not properly formatted. This makes it so that if `golint` fails,`gofmt` can be run to fix this kind of failure.